### ISSUE TITLE
[bdo.GUIloader] remove FTP download for Mega links

### ISSUE THREAD

**Issue by netzulo** [r3peat/R3p.BDO#12]
# Description

Just use this Nugget package : https://gpailler.github.io/MegaApiClient/articles/samples.html#download-file-from-a-mega-link

``` CSharp
void Main()
{
  var client = new MegaApiClient();
  client.LoginAnonymous();

  Uri folderLink = new Uri("https://mega.nz/#F!e1ogxQ7T!ee4Q_ocD1bSLmNeg9B6kBw");
  IEnumerable<INode> nodes = client.GetNodesFromLink(folderLink);
  foreach (INode node in nodes.Where(x => x.Type == NodeType.File))
  {
    Console.WriteLine($"Downloading {node.Name}");
    client.DownloadFile(node, node.Name);
  }

  client.Logout();
}
```

## SPECs 

+ [ ] Remove FTP connection from all projects 
+ [ ] Add Nugget for Mega download links
+ [ ] Allow to customize Mega link from Settings
+ [ ] Unzip files and **refactor start**

**Comment by netzulo** [comment]
Let me fix this

**Comment by Acen** [comment]
Why prefer a Mega folder rather than the current implementation using FTP?

This is getting super interesting though, having someone else join in with changes. I might jump in as well.

**Comment by netzulo** [comment]
umm... then.. we will change FTP to be more usable , some lanes at README about how to get working your own FTP ? @Acen :)

**Comment by Acen** [comment]
Potentially having some sort of application settings file?
Something simple like 
```
updates: [
enabled: true,
method: "ftp",
],
```

**Comment by netzulo** [comment]
fill work at this tomorrow :) README lanes too! ^^ but we need to know what think about your solution @r3peat

**Comment by netzulo** [comment]
then for this issue concept will be fixed at settings on issue #13
